build.ps1

```diff
@@ -38,24 +38,30 @@ if($MSBuildFullPath -eq "")
 {
     if (Test-Path "env:\ProgramFiles(x86)")
```

Comment:
What is the point of this? A lot of machines have both an x86 and non x68 program file path. Should this be inverted to look like 

if (programfiles) use programfiles
else if(programfilesx86) use programfilesx86